app/controllers/pipeline_outputs_controller.rb

```diff
@@ -32,6 +32,7 @@ def show_taxid_fasta
       nt_array = get_taxid_fasta(@pipeline_output, params[:taxid], params[:tax_level].to_i, 'NT').split(">")
       nr_array = get_taxid_fasta(@pipeline_output, params[:taxid], params[:tax_level].to_i, 'NR').split(">")
       @taxid_fasta = ">" + ((nt_array | nr_array) - ['']).join(">")
+      @taxid_fasta = "Coming soon" if @taxid_fasta == ">" # Temporary fix
```

Comment:
Maybe you dont need this line if you do get ‘’. It just means it has no data? If it’s a key error, you will actually get coming soon.

Reply:
Ok, I moved 'coming soon' code to pipeline_outputs_helper in order not to have these temporary fixes in too many different places. @taxid_fasta should never be empty though, because if the taxid shows up in the report, it should have at least one fasta record (we're displaying the union of NT and NR now)